packages/xs-extra/xapi-storage.master/opam

```diff
@@ -1,25 +1,21 @@
 opam-version: "1.2"
+name: "xapi-storage"
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://xapi-project.github.io/"
 bug-reports: "https://github.com/xapi-project/xapi-storage/issues"
 dev-repo: "git://github.com/xapi-project/xapi-storage.git"
-build: [
-  [make]
-]
-install: [
-  [make "install" "PREFIX=%{prefix}%"]
-]
-remove: [
-  [make "uninstall" "PREFIX=%{prefix}%"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
+  "jbuilder" {build}
+  "ounit" {test}
+  "cow"
   "rpc"
-  "re"
+  "caml2html"
```

Comment:
Can this be made an optional dependency? We can still build the whole package without it, its just generating the docs that requires it.